prime-angular-frontend/src/app/modules/adjudication/pages/site-banner-list-page/site-banner-list-page.component.ts

```diff
@@ -37,6 +37,6 @@ export class SiteBannerListPageComponent implements OnInit {
 
   private getBanners(): void {
     this.bannerResource.getSiteLandingBanners()
-      .subscribe((banners: Banner[]) => this.banners = banners);
+      .subscribe((banners: Banner[]) => this.banners = banners.reverse());
```

Comment:
Though we can do this in the web client.  It would be better to have the order determined in the API.  There are two pages that get banners based on location in adjudication - sites and enrolments.  So the order can be determined within the querying of `GetBannersAsync`

Reply:
Sure, I will reverse that in the API then.

Reply:
Moved